turbopuffer/namespace.py

```diff
@@ -38,7 +38,7 @@ class AttributeSchema:
     The schema for a particular attribute within a namespace.
     """
 
-    type: str # one of: '?string', '?uint', '[]string', '[]uint'
+    type: str # one of: 'string', 'uint', '[]string', '[]uint'
```

Comment:
probably should just delete this comment, it's not a doc comment, and I forgot to update it to add uuid